README.md

```diff
@@ -9,8 +9,10 @@ These images build and publish automatically using GitHub Actions.
 Image name | Description | Source Dockerfile
 ---------- | ----------- | -----------------
 `iree-org/amdgpu_ubuntu_jammy_x86_64` | Ubuntu with AMDGPU deps | [Source](./dockerfiles/amdgpu_ubuntu_jammy_x86_64.Dockerfile)
+`iree-org/amdgpu_ubuntu_ghascale` | Ubuntu with AMDGPU deps (GitHub Runner Scale Set) | [Source](./dockerfiles/amdgpu_ubuntu_ghascale.Dockerfile)
 `iree-org/amdgpu_ubuntu_jammy_ghr_x86_64` | Ubuntu with AMDGPU deps (GitHub runner) | [Source](./dockerfiles/amdgpu_ubuntu_jammy_ghr_x86_64.Dockerfile)
```

Comment:
It can be a follow-up change, but I think we should also prune old and no longer used dockerfiles from this repository:

* `amdgpu_ubuntu_jammy_x86_64`
* `amdgpu_ubuntu_jammy_ghr_x86_64`
* `cpubuilder_ubuntu_jammy_ghr`
* `manylinux_ghr_x86_64`

If these new dockerfiles don't use https://github.com/iree-org/base-docker-images/tree/main/docker-github-actions-runner (instead inheriting from `ghcr.io/actions/actions-runner:latest`), we could delete that also.

Reply:
Per my later comment, I think we should actually keep `cpubuilder_ubuntu_jammy_ghr` and merge it with the new `cpubuilder_ubuntu_ghascale`. It seems like we can drop the docker-github-actions-runner folder and inherit from `ghcr.io/actions/actions-runner` instead of `docker.io/myoung34/github-runner` now though.